packages/utils/src/abi_decoder.ts

```diff
@@ -1,9 +1,10 @@
-import { BigNumber } from '@0xproject/utils';
+import { AbiType, DecodedLogArgs, LogWithDecodedArgs, RawLog, SolidityTypes } from '@0xproject/types';
 import * as _ from 'lodash';
 import * as Web3 from 'web3';
 import * as SolidityCoder from 'web3/lib/solidity/coder';
 
-import { AbiType, ContractEventArgs, DecodedLogArgs, LogWithDecodedArgs, RawLog, SolidityTypes } from '../types';
+// tslint:disable-next-line:no-unused-variable
```

Comment:
Can you add a line explaining why this tslint is being ignored here.

Reply:
I wish I knew that. Looks like a tslint bug, cause the import is clearly not unused.

Reply:
There are some other linter false-positives that I'm investigating right now